Documentation/features-per-distro.md

```diff
@@ -48,7 +55,7 @@ This list shows a release corresponding to "full".
 - Extra Elements and Subplugins
     - Enabled
         - tensorflow-lite (1.13)
-        - tensorflow2-lite (2.3) w/ gpu delegation
+        - tensorflow2-lite (2.3, 2.7.0) w/ delegation support (GPU, XNNPACK, NNAPI, ...) since 2.7.0
```

Comment:
1. We now support 2.8.1 as default.
2. How about removing ver 1.13 ?

Reply:
Updated!